### DISCUSSION TITLE
Emoji support.

### DISCUSSION THREAD

**Discussion by ThreeTheOG** [dailydotdev/daily#1665] [Feature Requests]
It would be cool to see emoji support in the titles of posts. I saw a post the other day that had `:robot:` in it and think it would be cool if that was supported instead of it just being plain text.

**Comment by idoshamun** [comment]
Hi @ThreeTheOG,

We avoid emojis in titles as they're very much used for clickbait purposes.
We will add support in the post body or comments in the future, but not in the title.
